### ISSUE TITLE
Can't add plugin to consumer

### ISSUE THREAD

**Issue by bladefist** [PGBI/kong-dashboard#144]
## Issue summary

If you go to the plugins for a consumer, there is an "Add Plugin" button. This is on the `/#!/consumers/CONSUMER-ID/plugins` page.  

If you click Add Plugin, it drops the consumer ID though, and the plugin you add goes to ALL, not the consumer you were previously on.

I'm not sure if this is a bug or a feature request.  But it would be awesome to be able to add a plugin to a specific consumer via the portal.  Thanks!

**Comment by peeyush1234** [comment]
@bladefist You can add a plugin to a specific consumer. After you select a plugin via the portal, you get an option to select which consumer you want to add the plugin to. Let me know if this helps.

**Comment by bladefist** [comment]
@peeyush1234 you can but if you have thousands of consumers, it wont be in the dropdown list.  For this reason, you sometimes need to find the consumer first, and then add the plugin from there.

**Comment by peeyush1234** [comment]
@bladefist Makes sense. Also, functionality like typeahead can also solve this problem. Using typeahead, you can search for a specific consumer before adding a plugin to it.
